src/main/java/seedu/recipe/storage/ExportManager.java

```diff
@@ -0,0 +1,74 @@
+package seedu.recipe.storage;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+
+/**
+ * API to export current RecipeBook
+ */
+public class ExportManager {
+
+    private final Path recipeBookFilePath = Paths.get("data", "recipebook.json");
```

Comment:
default file path should not be hard coded to `data/recipebook.json` and should instead use `userPrefs.getRecipeBookFilePath()`

refer to `StorageManager.saveRecipeBook(recipeBook)` to see how this is done

Reply:
Very good suggestion

Reply:
Still not resolved - mark a new issue or PR to fix this AND the import/export functions into their own classes